orc8r/cloud/deploy/orc8r_deployer/docker/root/scripts/playbooks/roles/services/orc8r/tasks/main.yml

```diff
@@ -79,4 +79,4 @@
   tags: verify_sanity
   with_items:
     - magma/v1/networks
-    - magma/v1/lte
```

Comment:
Seems like a whitespace change. Can you remove this change.

Reply:
sure